utils/frame/remote-externalities/src/lib.rs

```diff
@@ -78,47 +111,32 @@ pub struct OfflineConfig {
 
 /// Description of the transport protocol (for online execution).
 #[derive(Debug, Clone)]
-pub enum Transport {
+pub struct Transport {
 	/// Use the `URI` to open a new WebSocket connection.
-	Uri(String),
+	uri: Option<String>,
```

Comment:
why introduce the `Option's`?

Reply:
Right, does not make sense 🙈 

Note the main reason I did this is because after creating the ws-client, I want to still know from which uri was used, something that ironically I cannot do with `WsClient`, ergo change to struct.

Reply:
But, we won't need any of that anymore because we don't create a new client per thread, so I will revert all these changes.